articles/aks/troubleshooting.md

```diff
@@ -28,6 +28,37 @@ You need to [request cores](https://docs.microsoft.com/azure/azure-supportabilit
 The maximum pods-per-node setting is 30 by default if you deploy an AKS cluster in the Azure portal.
 The maximum pods-per-node setting is 110 by default if you deploy an AKS cluster in the Azure CLI. (Make sure you're using the latest version of the Azure CLI). This default setting can be changed by using the `–-max-pods` flag in the `az aks create` command.
 
+## Is it safe if the IP address ranges overlap with one another or with on-premise networks?
+
+Overlapping subnets and IP ranges could cost several days of lost productivity if issues come up because of deviation from recommended practices.
+ 
+This could help shed some light on some of the issues one could encounter as a result of such overlaps: 
+ 
+In advanced networking scenarios, the IP address plan for an AKS cluster consists of the following partitions:
+ 
+1.	virtual network
+2.	at least one subnet (for nodes, pods Kubernetes and Azure resources)
+3.	a Kubernetes service address range.
+4.	Range of ips for a docker bridge
+
+The subnet(s) in (2) are partitions from the virtual network (1) and is used to allocate addresses to nodes, pods, and internal ingress or load balancer ips. Advanced planning is necessary to ensure that it will accommodate or fit in all the nodes, pods, and all Kubernetes and Azure resources that might be provisioned in your AKS cluster.
+ 
+The service address range in (3) is where ip for services will be allocated from. The Kubernetes DNS IP address used by cluster service discovery (kube-dns) will also come from (3).
```

Comment:
Core-dns
not
kube-dns

Reply:
It's not clear to me from here if it's fine to collide/be re-used in other clusters or not.

Answer yes, it can be re-used across clusters, but ideally should not collide with peered networks and on-prem, since if a pod tries to access a resource there, the system might not be able to decide correctly if it should be send the traffic to the peered network or to a service.